apis/datadoghq/v1alpha1/datadogdashboard_types.go

```diff
@@ -50,6 +50,9 @@ type DatadogDashboardSpec struct {
 	// +listMapKey=id
 	// +optional
 	Widgets []Widget `json:"widgets,omitempty"`
+	// String that represents the whole dashboard.
```

Comment:
Can you provide more details about what it is. Something like json string representation of Datadog API dashboard object.